scripts/test.sh

```diff
@@ -11,3 +11,4 @@ version="1.$minor.x"
 
 export KUBEBUILDER_ASSETS=$(setup-envtest use -p path "$version")
 go test ./...
+# go test --race ./...
```

Comment:
Why is this being added as a comment?

Reply:
Because I wanted to hop between race and non-race mode and forgot to throw out that change :)